static/app/views/eventsV2/homepage.spec.tsx

```diff
@@ -311,7 +311,9 @@ describe('Discover > Homepage', () => {
     expect(screen.queryByText('14D')).not.toBeInTheDocument();
   });
 
-  it('renders changes to the discover query when no homepage', async () => {
+  // Disabling because it has been flaky, will fix in DD-1151
+  // eslint-disable-next-line jest/no-disabled-tests
+  it.skip('renders changes to the discover query when no homepage', async () => {
```

Comment:
please include the ticket number here -- https://github.com/getsentry/rfcs/blob/main/text/0027-manual-disabling-of-flaky-tests.md

Reply:
Done, thanks!